cvc5_z3py_compat/z3.py

```diff
@@ -6106,16 +6192,17 @@ def is_fprm_value(a):
 
 
 # Global default rounding mode
-_dflt_rounding_mode = RTZ()
+_dflt_rounding_mode = pc.RoundingMode.RoundTowardZero
```

Comment:
Isn't `RoundTowardZero` outdated? The kinds and types should now have the same names as the C++ enums.

Reply:
Hm. I don't know (and don't care, to be honest). The important change is that this `_dflt_...` don't store the wrapper objects (that have a handle on the solver), but only the internal enum. What do you think it should be?

Reply:
Ok this didn't happen yet on the cvc5 side, but will be soon `ROUND_TOWARD_ZERO` (the same name as in c++).